packages/engine/Source/Core/Check.d.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * Contains functions for checking that supplied arguments are of a specified type
+ * or meet specified conditions
+ */
+export const Check: {
+  /**
+   * Throws if test is not defined
+   *
+   * @param {string} name The name of the variable being tested
+   * @param {*} test The value that is to be checked
+   * @exception {DeveloperError} test must be defined
+   */
```

Comment:
Can you explain why the inline docs need to be duplicated?

Reply:
Without them the intellisense that VSCode picks up has no description of any of the functions. The way the build is adding this it's taking the file wholesale. I was not able to find a way to get the `tsd-jsdoc` to output anything other than `const Check: any` so it also didn't include any of the descriptions other than the top one for `Check`. It's annoying and I'd love to remove the duplication but I couldn't figure out a way to do that and I didn't want to spend a ton more time on digging into it.